mst/page-index/page-index.js

```diff
@@ -11,6 +11,19 @@ module.exports = [
         title: "About Managed Service for TimescaleDB",
         href: "about-mst",
       },
+      {
+        title: "Command Line tool for Managed Service for TimescaleDB",
+        href: "aiven-client",
+        children: [
+        {
+        title: "Aiven Client for Managed Service for TimescaleDB",
+        href: "aiven-client-install",
+        },
+        {
+        title: "Create a fork using Aiven Client",
+        href: "create-fork",
+        },
+       ],
```

Comment:
I think there is a missing curly brace here, can you indent these like the other entries? It makes it really hard to check bracket closing if they aren't indented 🙏🏼

Reply:
preview is returning 403 for me ...

Reply:
<img width="886" alt="Screenshot 2022-08-29 at 8 00 07 AM" src="https://user-images.githubusercontent.com/37059749/187111293-3cdfe263-92d8-4fe2-ab71-0e5f71c769a9.png">
Same here!!!

Reply:
I'm looking at it now, Rajie.

Reply:
I have it working as a single page:

![image](https://user-images.githubusercontent.com/3914967/187119122-2f252549-8900-471e-aca4-31fe6689a8c0.png)

We can either merge like this (if you want), or I can keep trying to get it into a nested page after lunch 🥪

Reply:
Let's merge it @Loquacity  considering that we have three nested pages, install cli, fork service, and auth plugin pages. We can may be have a new PR for nested pages after the new pages are merged. Thank you so much @Loquacity .

Reply:
np. we can work it out with the next PR 👍